src/Components/IndividualReview.jsx

```diff
@@ -17,6 +18,16 @@ export const IndividualReview = () => {
     });
   }, []);
 
+  const HandleUpvotes = () => {
+    const newReview = [...indReview];
+    newReview[0].votes++;
+    setIndReview(newReview);
+    patchVotes(review_id).catch((err) => {
+      setIndReview(newReview[0]--);
```

Comment:
I'm not too sure what's happening here. I thought newReview[0] was an object you were storing in state, so we can't '--' it.